tests/integration/devtools/test_callback_validation.py

```diff
@@ -730,11 +730,7 @@ def multipage_app(validation=False):
     layout_page_2 = html.Div(
         [
             html.H2("Page 2"),
-            dcc.Dropdown(
```

Comment:
`dcc.Dropdown` is giving React deprecated lifecycle method warnings ATM... I thought we had already updated all of these in the 16.13 push? Anyway switched to `Input` to get my `assert not dash_duo.get_logs()` clean console test to pass.